File: python/grepc.py
#!/usr/local/bin/python

import re, sys, time

t0 = time.time()
counter = 0
with open(sys.argv[2]) as f:
	for line in f:
		if re.search(sys.argv[1], line):
			counter += 1
print counter
t1 = time.time()
print t1-t0

File: perl/grepc.pl
#!/usr/local/bin/perl

use Time::HiRes qw(time);

$t0 = time;
$counter = 0;
open(F, '<', $ARGV[1]);
while ($line = <F>) {
	if (index($line, $ARGV[0]) != -1) {
		$counter++
	}
}
close(F);
print "$counter\n";
$t1 = time;
print ($t1-$t0);
print "\n";

